languages.toml

```diff
@@ -121,6 +121,9 @@ scope = "source.json"
 injection-regex = "json"
 file-types = ["json"]
 roots = []
+language-server = { command = "vscode-json-language-server", args = ["--stdio"] }
```

Comment:
How is this normally installed? I see the [vscode-json-languageserver](https://www.npmjs.com/package/vscode-json-languageserver) but the binary is not the same name